### PR TITLE
Improve authorizer validation

### PR DESCRIPTION
**Pull request by srvance**
Addresses issue #1129. Also, added `.gitignore` entries for JetBrains products.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,7 @@ Session.vim
 *~
 # auto-generated tag files
 tags
+
+### JetBrains files ###
+.idea/
+*.iml
```
Comment (marcoow) [inline .gitignore]: I think this is sth. that people should have in their global `.gitignore` really.

Comment (srvance) [inline .gitignore]: Done. I was surprised to see the vim entries there.

**Comment by marcoow** [comment]
Yeah, those should also be removed actually…

**Comment by srvance** [comment]
Want me to remove those and push while I'm at it or would you like it accounted for separately?

**Comment by marcoow** [comment]
sure, that'd be great - should be a separate PR though

**Comment by marcoow** [comment]
awesome - thanks!

**Pull request merged**